.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Probably should add this to the .gitignore, as it's related more to your own IDE instead of the project.